packages/qwik-city/adapters/vercel-edge/vite/index.ts

```diff
@@ -131,6 +134,14 @@ export interface VercelEdgeAdapterOptions extends ServerAdapterOptions {
    * come from a static file, rather than a server-side rendered response.
    */
   staticPaths?: string[];
+
+  /**
+   * Define the `target` proeprty in the `ssr` property in the `vite.config.ts` file.
+   *
+   * Defaults to `webworker` for not having a breaking change. But `node` will become the default
```

Comment:
I need some clarification here. How can `edge` have `node` as the default? I would expect that edge is `webworker` API not `node` API.

Reply:
I understand that it's confusing, but `webworker` does not fit 100% too, right? `process.env` exists on edge, but not on webworker environments. we have Buffer on global. We have some Node.js module support. It's not completely Node.js and not completely webworker, but something in between <picture data-single-emoji=":thonk:" title=":thonk:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thonk/d620c05f0e664ad6.png" alt=":thonk:" align="absmiddle"></picture>

Reply:
In that case, would it not be better to set it to `webworker` and than add the missing parts from `node`. My worry is that if we make it `node`, than things that should be valid (because you think you have node) are not.

If I understand this correctly, this switch controls the `.d.ts` files used for compilation and it will under-restrictive.